src/chunk.c

```diff
@@ -2766,18 +2766,29 @@ init_scan_by_compressed_chunk_id(ScanIterator *iterator, int32 compressed_chunk_
 }
 
 bool
-ts_chunk_contains_compressed_data(Chunk *chunk)
+ts_chunk_contains_compressed_data(Chunk *chunk, Chunk **parent)
```

Comment:
that function name is not what the function does, it seems to check if chunk is a compressed chunk and looks up the parent if it is

Reply:
ts_chunk_is_compressed would make more sense

Reply:
or ts_chunk_is_compressed_chunk

Reply:
Any suggestions on a better name? :)

Reply:
Problem is that it is not clear what compressed chunk actually means here. Is it a main chunk or chunk associated with compressed data. Also to note, that this function is used in other places as well